code/game/objects/structures/aliens.dm

```diff
@@ -144,30 +144,33 @@
 	smoothing_flags = SMOOTH_BITMASK
 	smoothing_groups = list(SMOOTH_GROUP_ALIEN_RESIN, SMOOTH_GROUP_ALIEN_WEEDS)
 	canSmoothWith = list(SMOOTH_GROUP_ALIEN_WEEDS, SMOOTH_GROUP_WALLS)
-	var/last_expand = 0 //last world.time this weed expanded
-	var/growth_cooldown_low = 150
-	var/growth_cooldown_high = 200
-	var/static/list/blacklisted_turfs
-
+	///the parent node that will determine if we grow or die
+	var/obj/structure/alien/weeds/node/parent_node
```

Comment:
if there are multiple weed nodes near each other, the weeds will all be destroyed even though there is a node in range

Reply:
only the child weeds, which is intentional.